main.go

```diff
@@ -188,6 +193,9 @@ func updatePage(
 			"number":    nextPageVersion,
 			"minorEdit": false,
 		},
+		"ancestors": []map[string]interface{}{
+			{"id": pageInfo.Ancestors[len(pageInfo.Ancestors)-1].Id},
```

Comment:
please move this stuff to variable like `nextPageVersion` and check for len(pageInfo.Ancestors) > 0